pytest_fixtures/component/smartproxy.py

```diff
@@ -32,6 +31,6 @@ def module_fake_proxy(request, module_target_sat):
             args['url'] = url
             proxy = module_target_sat.api.SmartProxy(**args).create()
             yield proxy
-            capsule_cleanup(proxy.id)
+            target_sat.cli.Proxy.delete({'id': proxy.id})
```

Comment:
You can use `module_target_sat` here, no need to include both.

Reply:
Ah, I thought I probably could.